scripts/fast-reboot

```diff
@@ -281,7 +281,7 @@ fi
 # Kill bgpd to start the bgp graceful restart procedure
 debug "Stopping bgp ..."
 docker exec -i bgp pkill -9 zebra
-docker exec -i bgp pkill -9 bgpd
+docker exec -i bgp pkill -9 bgpd || [ $? == 1 ]
```

Comment:
|| true would be enough?

Reply:
|| true will white list all errors. pkill return code 1 means process not found. We should be ok with this failure.

However, if the BGP is supposed to be configured, then we should not get into warm-reboot at the first place. This is for warm-reboot pre-check.